package.json

```diff
@@ -68,7 +68,8 @@
     "axios": "^0.28.0",
     "axios-better-stacktrace": "^2.1.6",
     "axios-retry": "^3.3.1",
-    "date-fns": "^2.29.3",
+    "date-fns": "^3.6.0",
+    "date-fns-tz": "^3.1.3",
```

Comment:
Using external library to support proper timezoning on cross devices. There are some differences in the way JS engines handle timezone converting. date-fns-tz is confirmed and battle tested with the Hermes engine

Reply:
Btw just a note that [this issue](https://github.com/AtB-AS/kundevendt/issues/7795) becomes possible to complete after upgrading to date-fns v3.

Reply:
👍 Only handle necessary breaking changes in this PR, but I guess this can be relevant for in relation to https://github.com/AtB-AS/kundevendt/issues/16313 @jorelosorio ? Or maybe better yet separate PR